### DISCUSSION TITLE
Welcome to telegram-bot-spring Discussions!

### DISCUSSION THREAD

**Discussion by nazarovctrl** [nazarovctrl/telegram-bot-spring#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akbarjorayev** [comment]
Create 7th version of telegram-bot-spring

**Comment by akbarjorayev** [comment]
Then 8th version of telegram-bot-spring
